### ISSUE TITLE
Near tab

### ISSUE THREAD

**Issue by NIK916** [horosproject/horos#390]
Why I can't see near tabs during my studies? I need  to compare pathological signs during my MR study. There are items to check that I didn't found?
If I press cmd + click on new sequence, my horos opens single tabs. 
Thanks to your reply.
All the best
Dr. Nicola Della Camera

**Comment by PurviewSierra** [comment]
Hello Dr. Nicola, 
Please select all studies you need to view and then select 'Windows' from the toolbar (see screenshot). From there you should be able to select the amount of tabs you would like open. 

Please let me know if you need further clarification. 

Regards, 
Sierra 

![screen shot 2018-09-14 at 12 34 51 pm](https://user-images.githubusercontent.com/43280486/45571340-c0e9ac00-b81a-11e8-9ee1-de179355a17f.png)

**Comment by NIK916** [comment]
Dear Sierra,
Thanks to your reply. Usually I use horos in other Mac and so I know that icon to put nearly tabs. I have done a picture to show you much better my problem.

All the best

**Comment by NIK916** [comment]
Hello Sierra, 
I’m waiting for your support! Please help me because I need to these informations to end an important MR study!
All the best
Nicola

**Comment by JeanClaude2** [comment]
Did you try with [OsiriX](https://www.osirix-viewer.com/) ?

**Comment by NIK916** [comment]
No, I’ve tried to reply to your colleague but I didn’t have answer

**Comment by NIK916** [comment]
I try just now to use Osiris and tabs are open in correct view.
I think that the problem is in some point in my Horos' setting.

**Comment by PurviewSierra** [comment]
Hello Dr. Nicola,
Are you able to attach a screenshot of what you're seeing incorrectly in Horos and the correct view you are seeking in Osirix?

Regards, 
Purview Customer Support

Need a faster response? [Contact our support team](https://horosproject.org/professional-support/)!

**Comment by NIK916** [comment]
This is my problem with horos

**Comment by PurviewSierra** [comment]
Hello Dr. Nicola, 
I apologize however I do not see any screenshots attached or inserted into your message. 

Regards, 
Purview Customer Support
